### DISCUSSION TITLE
pro active chat invitation - show invitation limit not working

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1846] [General]
i found a problem in pro active chat invitation with `How many times user show invitation, 0 - untill users closes it, > 0 limits.`. i would expect that after defining the number e.g. 2 the invitation is shown only 2 times, but it's shown all the time after multiple times refreshing the browser regardless of setting up this value. perhaps a bug?
![image](https://user-images.githubusercontent.com/13809377/207532882-7d79e69b-7b35-4b4b-af3f-b5788d64aee0.png)

**Comment by remdex** [comment]
Fixed
